doc/getting_started/index.rst

```diff
@@ -167,20 +167,36 @@ Get Zephyr and install Python dependencies
 
 Next, clone Zephyr and its :ref:`modules <modules>` into a new :ref:`west
 <west>` workspace named :file:`zephyrproject`. You'll also install Zephyr's
-additional Python dependencies.
+additional Python dependencies. West and the rest of the Python dependencies
+will be installed in a Python `virtual environment
+<https://docs.python.org/3/tutorial/venv.html>`_ which you'll create inside the
+workspace.
 
 .. tabs::
 
    .. group-tab:: Ubuntu
 
-      #. Install west, and make sure :file:`~/.local/bin` is on your
-         :envvar:`PATH` :ref:`environment variable <env_vars>`:
+      #. Create and switch to a new virtual environment:
 
          .. code-block:: bash
 
-            pip3 install --user -U west
-            echo 'export PATH=~/.local/bin:"$PATH"' >> ~/.bashrc
-            source ~/.bashrc
+            python3 -m venv --prompt zephyrproject ~/zephyrproject/venv
```

Comment:
At least on Ubuntu 18.04 LTS, this requires the `python3-venv`(or `python3.8-venv`) package to be installed.

Reply:
> , this requires the `python3-venv`(or `python3.8-venv`) package to be installed.

I added python3-venv, thanks.